docs/v3.x/admin-panel/deploy.md

```diff
@@ -32,64 +29,22 @@ module.exports = {
 
 ## Deploy the administration panel on another server (AWS S3, Azure, etc) than the API.
 
-It's very common to deploy the front-end and the back-end on different servers. Here are the required configurations to handle this case:
+It's very common to deploy the front-end and the back-end on different servers. Here is the required configuration to handle this case:
```

Comment:
We need to note that normal `url` param here will not actually prefix things in Strapi and requires the use of some other 3rd party proxy application.